lls/cache.c

```diff
@@ -471,6 +494,12 @@ process_icmp_pkts(struct lls_config *lls_conf, struct lls_icmp_req *icmp)
 			continue;
 		}
 
+		if (icmphdr->icmp_type == ICMP_MASK_REQUEST_TYPE &&
+				icmphdr->icmp_code == ICMP_MASK_REQUEST_CODE) {
+			ping_pkts[num_ping_pkts++] = pkt;
+			continue;
+		}
+
```

Comment:
The body of this `if` statement is identical to the one above, so just combine the tests with an `||`.